boards/arm/gd32f470i_eval/board.cmake

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2022, Rtone.
+# SPDX-License-Identifier: Apache-2.0
+
+board_runner_args(jlink "--device=GD32F450IK" "--speed=4000")
```

Comment:
this looks wrong

Reply:
I've seen the comment: please add a comment mentioning segger doesn't support f470 yet but that this works (?)